ansible/roles/aws_docker_setup/tasks/minimal.yml

```diff
@@ -49,3 +49,8 @@
         name: docker
         enabled: yes
         state: restarted
+
+    - name: "Open permissions on docker socket (needed until 'meta: reset_connection' is supported)."
```

Comment:
Note that these permissions are locked back down at the end of provisioning, see below.

Reply:
@djwhatle could we add a section in ec2/README.md to document this?